client/src/components/advancedSearch/TaskFilter.tsx

```diff
@@ -78,6 +84,188 @@ const TaskFilter = ({
 
   const parentKey = "parentTask"
   const valueKey = multi ? "uuid" : "shortName"
+
+  const taskFields = `
```

Comment:
You should move this to the top, since you need:
```javascript
const GQL_GET_TASK = gql`
  query ($uuid: String!) {
    task(uuid: $uuid) {
      ${taskFields}
    }
  }
`

const GQL_GET_TASKS = gql`
  query ($uuids: [String]) {
    tasks(uuids: $uuids) {
      ${taskFields}
    }
  }
`
```
else showing a saved search doesn't work correctly.